spacy/schemas.py

```diff
@@ -517,3 +517,6 @@ class DocJSONSchema(BaseModel):
     _: Optional[Dict[StrictStr, Any]] = Field(
         None, title="Any custom data stored in the document's _ attribute"
     )
+    user_data: Optional[Dict[StrictStr, Dict[StrictStr, Any]]] = Field(
```

Comment:
The schema wasn't updated but all the tests pass. There should be a schema test that is expected to pass with all types of extensions.

Reply:
What would be the best way to write a unit test for that? Since these fields are `Optional`?